src/code/z_prenmi.c

```diff
@@ -16,21 +16,21 @@ void PreNMI_Update(PreNMIContext* prenmiCtx) {
     prenmiCtx->timer--;
 }
 
-#ifdef NON_MATCHING
-// Minor reordering around call to func_8012C470
 void PreNMI_Draw(PreNMIContext* prenmiCtx) {
     GraphicsContext* gfxCtx = prenmiCtx->state.gfxCtx;
 
     func_8012CF0C(gfxCtx, 1, 1, 0, 0, 0);
+
+    OPEN_DISPS(gfxCtx);
+    
     func_8012C470(gfxCtx);
 
-    gDPSetFillColor(gfxCtx->polyOpa.p++,
-                    (GPACK_RGBA5551(0xFF, 0xFF, 0xFF, 1) << 16) | GPACK_RGBA5551(0xFF, 0xFF, 0xFF, 1));
-    gDPFillRectangle(gfxCtx->polyOpa.p++, 0, prenmiCtx->timer + 100, 320 /*SCREEN_WIDTH*/ - 1, prenmiCtx->timer + 100);
+    gDPSetFillColor(POLY_OPA_DISP++,
+                    (GPACK_RGBA5551(255, 255, 255, 1) << 16) | GPACK_RGBA5551(255, 255, 255, 1));
+    gDPFillRectangle(POLY_OPA_DISP++, 0, prenmiCtx->timer + 100, 320 /*SCREEN_WIDTH*/ - 1, prenmiCtx->timer + 100);
```

Comment:
I know it wasn't you, but why not use `SCREEN_WIDTH` here?
```suggestion
    gDPFillRectangle(POLY_OPA_DISP++, 0, prenmiCtx->timer + 100, SCREEN_WIDTH - 1, prenmiCtx->timer + 100);
```